Enhancement/run app via package installation (#172)
pharmaverse/aNCA: README.md

* **Save your analysis settings** and reupload them later to keep on analysing!

## Installation
### Via pak (recommended)
We recommend using [pak](https://github.com/r-lib/pak) for package installation, along with all system dependencies. If you do not have `pak` available, you will need to set it up first:
```R
install.packages("pak")
```
then you can install  [aNCA](.) by running:
```R

> pak::install("pharmaverse/aNCA")

Reviewer: issue: readme instructions with a small typo
```suggestion
pak::pkg_install("pharmaverse/aNCA")
```

```
in your R console.

### Via cloning the repository (for contributors)

> Alternatively, you can set up the package by cloning the repository:

Reviewer: nitpick: I know it may seem obvious, but last time when I gave these instructions I realized many people got lost because they did not notice these actions are not in the same space (terminal / ide). I would suggest if ok to also include it in the text so there is no doubt:

```suggestion
Alternatively, you can set up the package by cloning the repository through your terminal/shell:
```

```bash
git clone https://github.com/pharmaverse/aNCA.git
```

> and then loading it directly using [devtools](https://github.com/r-lib/devtools):

Reviewer: nitpick: same thing here
```suggestion
and then loading it using [devtools](https://github.com/r-lib/devtools) in your IDE (e.g, RStudio):
```
